packages/core/src/components/Dialog/Dialog.tsx

```diff
@@ -140,6 +142,22 @@ export const HvDialog = ({
     ? withTooltip(closeButtonDisplay, buttonTitle, "top")
     : closeButtonDisplay;
 
+  const slots = useMemo<ModalSlots>(
```

Comment:
`ModalSlots` is from `@mui/base` which isn't a direct dependency or peer-dependency.
`MuiDialogProps["slots"]` is more future-proof and is already imported

```suggestion
  const slots = useMemo<MuiDialogProps["slots"]>(
```

Reply:
Resolved on my last update 🚀